index.html

```diff
@@ -0,0 +1,152 @@
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
+                      <div class="btn3">
+                        <a href="image3.png" target="_blank">See Project</a>
+                      </div>
+            </div>    
+           </div>
+           <div class="container2">
+            <div class="imgdiv2">
+                <img class="projects" src="image4.png" alt="Nature">
+            </div>
+                   <div class="multi">
+                    <h1 class="popheading">Multi Post Series</h1>
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <br>
+                    <span class="htcj2">HTML</span> 
+                    <span class="htcj2">CSS</span>
+                    <span class="htcj2">Javascript</span><br><br>
+                      <div class="btn4">
+                        <a href="image4.png" target="_blank">See Project</a>
+                      </div> 
+                      </div>
+                    
+            </div>    
+           </div>
+           <div class="container5">
+            <div class="about">
+                <h1 class="abouth">About Myself</h1>
+                <p class="popp15">Hello I’m a software developer! I can help you build a product , feature or 
+                    website Look through some of my work and experience! If you like what you see and
+                     have a project you need coded, don’t hestiate to contact me.</p>
+                     <h3 class="pcol">LET’S CONNECT</h3>
+                <ul class="iconlist">
+                    <li><img class="icons" src="github.svg" alt="github"></li>
+                    <li><img class="icons" src="linkedinicon.svg" alt="linkedin"></li>
+                    <li><img class="icons" src="victory.svg" alt="victory"></li>
+                    <li><img class="icons" src="twitter.svg" alt="twitter"></li>
+                    <li><img class="icons" src="m.svg" alt="mimage"></li>
+                </ul>
+                <br><br>
+                         <span><a class="abouts" href="#">Get My Resume</a> </span> 
```

Comment:
- [ ] [REQUIRED] Please implement the previous reviewer's suggestion about all button styles 🙏🏼 `[HINT]` Implement to your buttons like this style, actually add: `border: 1px solid #DFE1E6;` `border-radius: 16px;`

#
![image](https://user-images.githubusercontent.com/63915024/213191803-97f664c0-de7f-4fb2-b72c-06594ec976c1.png)
#
![image](https://user-images.githubusercontent.com/63915024/213192563-663402f7-f541-445a-b4ee-3d593fd122b6.png)
#

Reply:
Design adjusted as per suggestions. Now sending for review again please approve its my third review.